### PR TITLE
perf - mark times for creating workbench contributions

### PR DESCRIPTION
**Pull request by bpasero**

### CODE REVIEW

File: src/vs/workbench/common/contributions.ts [hunk 1]
```diff
@@ -4,9 +4,10 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { IInstantiationService, IConstructorSignature, ServicesAccessor, BrandedService } from 'vs/platform/instantiation/common/instantiation';
```
Comment (huajiankan) [inline src/vs/workbench/common/contributions.ts:6]: ❤️

**Review by jrieken — DISMISSED** [review]

**Comment by bpasero** [comment]
@jrieken given layer breaker, I added `measure` to your `performance.js` but I was not able to find a polyfill because that method talks to the underlying browser devtools to mark an entry in the timeline...

**Review by jrieken — CHANGES_REQUESTED** [review]
Not a fan of this. We shouldn't have `measure` if we cannot polyfill it. It's all about being compatible in all environments: node, browser, worker etc

Also, measure isn't really needed. The absolute times can be seen in the perf editor and simple perf marks already show in dev tools

**Comment by bpasero** [comment]
@jrieken fair enough, I have removed `measure`

What I do like is this bar you get from it in `Performance` tab of devtools:

Screenshot 2022-08-30 at 14 19 55

Is there something comparable with the simple marks?

**Review by jrieken — APPROVED** [review]

**Comment by jrieken** [comment]
> Is there something comparable with the simple marks?

The small lines should be marks. At any time you add a measurement between two existing marks, e.g from the console or anywhere in the code but I don't think that the perf tool updates after the recording has stopped

**Comment by bpasero** [comment]
Oh yeah true, did not see them before!

**Pull request merged**